src/text.jl

```diff
@@ -77,7 +77,7 @@ function GtkTextIter(text::GtkTextBuffer, mark::GtkTextMark)
 end
 show(io::IO, iter::GtkTextIter) = print("GtkTextIter(...)")
 
-immutable GtkTextRange <: Range
+immutable GtkTextRange <: Range{Char}
```

Comment:
Exactly. As you surely noted, `getindex(::GtkTextRange, ::Int)` returns a `Char`, so `Char` it should be.